CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v2.2.4
```

Comment:
Do you mind bumping the `mix.exs` version as well?

Reply:
I bumped the version, thanks for reminding.